en/docs/services/create-your-first-service.md

```diff
@@ -1,168 +1,161 @@
 # Create Your First Service
 
-To learn how to create a service, let's try a simple scenario. In this example, you are creating a service that consumes a public API that fetches the current active COVID-19 cases in a specified country, testing it, deploying it, and then observing its performance.
+To learn how to create a service, let's look at a simple scenario. In this example, you'll create a service that consumes a public API that fetches the current active COVID-19 cases in the United States. Then you'll test it, deploy it, and observe its performance.
 
 ## Step 1: Create the service
 
-To create the service, follow the procedure below: 
+To create the service, follow this procedure: 
 
-1. Access the Choreo Console via https://console.choreo.dev/.
+1. Access the Choreo Console via https://console.choreo.dev/ and sign in using your Google or GitHub credentials.
```

Comment:
Shall we consider re-phrasing this to "Sign in to the Choreo Console at via https://console.choreo.dev/"?

Reply:
+1